### PR TITLE
[WIP] moved example query from readme to dir + steps to run queries

### PR DESCRIPTION
**Pull request by embidem**
also capitalized the headers because i couldn't help myself

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,22 +140,21 @@ machine. Each process is referenced by the port it is listening on and its
...
-SELECT hosts.NAME
-FROM ports
-  JOIN processes
-    ON ports.process_id = processes.id
-  JOIN hosts
-    ON processes.host_id = hosts.id
-WHERE ports.number = 6061
-  AND upper(ports.state) = "LISTEN"
-```
+Finding all of the hosts which are listening on a particular port: hosts_on_port.sql
```
Comment (DennyHoang) [inline README.md]: A single newline in Markdown does not actually split it onto separate lines.
( https://gist.github.com/shaunlebron/746476e6e7a4d698b373 )

**Review by DennyHoang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DennyHoang) [reply README.md]: @embidem

Comment (ohlookadollar) [reply README.md]: @embidem was this intended to be multiple lines?
Please advise

Comment (DennyHoang) [reply README.md]: Resolved in https://github.com/pivotal-cf/scantron/pull/8

File: README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+## Purpose
 
 Scantron is a tool for performing a census of processes, ports, protocols, and
 file permissions on VMs. Scans can be performed against single VMs, or against
 all VMs in a bosh deployment. The intention is to provide a point-in-time scan
 which can be analysed offline. Scans are stored in a SQLite database. The CLI
 provides a summary report format or the SQLite db can be queried directly.
 
-## usage
+## Use
```
[the remark this answers is not in the document]
Comment (DennyHoang) [reply README.md]: Resolved in https://github.com/pivotal-cf/scantron/pull/8

**Pull request merged**